client/human/chmain.mm

```diff
@@ -29,5 +29,11 @@ int main(int argc, char *argv[])
         return 0;   // quit without actually starting game
     }
 
+    // did the player request the version output?
+    if (GetOptionsDB().Get<bool>("version")) {
+        std::cout << "FreeOrionCH " << FreeOrionVersionString() << std::endl;
```

Comment:
You forgot to include `util/Version.h` here in `chmain.mm`, resulting of course in a compile error on OSX. I [created a PR](https://github.com/adrianbroher/freeorion/pull/3) for your branch with the fix.